ekncontent/ekncontent/eknc-domain.h

```diff
@@ -34,6 +35,7 @@ typedef enum {
   EKNC_DOMAIN_ERROR_BAD_RESULTS,
   EKNC_DOMAIN_ERROR_UNSUPPORTED_VERSION,
   EKNC_DOMAIN_ERROR_PATH_NOT_FOUND,
+  EKNC_DOMAIN_ERROR_ID_NOT_VALID,
```

Comment:
This is technically an API addition, so backporting to SDK1 would be problematic. @ebassi what do you think?